CHANGELOG.md

```diff
@@ -7,7 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-- Changed `btc::Network` (de)serializing into/from string
+## 0.8.1 - 2018-06-06
+
+### Internal improvements
+
+- Changed `btc::Network` (de)serializing into/from string (#128)
+
+- Updated to the [Rust-bitcoin 0.13.1](http`s://github.com/rust-bitcoin/rust-bitcoin/releases/tag/0.13)
```

Comment:
Seems like link is broken.
And it doesn't point to 0.13.1.

Reply:
Unfortunately link to version 0.13.1 doesn't exist

Reply:
Maybe then remove this link at all? (and btw I've been talking about ` in the link)